exasol_advanced_analytics_framework/lua/src/event_loop.lua

```diff
@@ -45,18 +45,17 @@ end
 --
 function M.init(query_to_event_handler)
     local status = "started"
-
+    local query_to_create_view = nil
     repeat
-        local success, result = _G.global_env.pquery(query_to_event_handler)
-        if not success then
-            local error_obj = exa_error.create(
-                    "E-AAF-2",
-                    "Error occurred in calling Event Handler: " .. result.error_message)
-            _G.global_env.error(tostring(error_obj))
-        end
-        query_to_event_handler = result[1][1]
-        status = result[2][1]
-        M._run_queries(result, 3)
+        -- call EventHandlerUDF return queries
+        local return_queries = {{query_to_create_view}, {query_to_event_handler}}
+        local result = M._run_queries(return_queries, 1)
+
+        -- handle EventHandlerUDF return
+        query_to_create_view = result[1][1]
+        query_to_event_handler = result[2][1]
+        status = result[3][1]
+        M._run_queries(result, 4)
     until (status == 'completed')
 
     return status -- TODO return
```

Comment:
this is a good point, when we get the status completed, the event handler should have the option to return a json object (Dict[str, any])